### DISCUSSION TITLE
PTZ Return to Home Position?

### DISCUSSION THREAD

**Discussion by Someguitarist** [blakeblackshear/frigate#7555] [Ask A Question] [answered]
Is there an easy way to have PTZ controls return the camera position to the same position it started at?  In my scenario, if I use the PTZ controls to move the camera my motion masks and stuff are off, unless I can return it t the exact position it started at.  Is there any easy way to say 'After 30 seconds, return to home position' that I'm missing?

Thanks!  Also, thanks for this great project!

**Comment by NickM-27** [comment, marked as answer]
> Is there any easy way to say 'After 30 seconds, return to home position' that I'm missing?

yes, just create a home preset and select that preset in the webui

**Comment by hawkeye217** [comment]
I'll create a PR to update the autotracking docs for even more clarity on this behavior for the autotracker.
